embc-app/ClientApp/src/app/volunteer/components/registration-maker/registration-maker.component.html

```diff
@@ -78,7 +78,7 @@ <h2 class="blue-header font-weight-bold">Task Number &amp; Place of Registration
         <div class="form-row">
           <app-form-field required="true" class="col">
             <label for="incidentTask">Task Number</label>
-            <select [compareWith]="compareById" [class.is-invalid]="invalid('incidentTask')" class="form-control"
+            <select [compareWith]="compareById" readonly disabled [class.is-invalid]="invalid('incidentTask')" class="form-control"
```

Comment:
I think there needs to be some logic here that only disables the select if the user is a volunteer/ERA user. All roles use this component to create a registration, but only the ERA user is locked into one Task Number per session.

Could you add some logic to only disable this control for ERA/Volunteers and leave as is for Superusers / admins? 

Thanks!